README.md

```diff
@@ -130,35 +130,45 @@ Once all of the required dependencies are installed and functioning, run:
 These commands will create:
  1. a `deephaven-core` directory containing the source code.
  2. Docker images containing everything needed to launch Deephaven.
- 
-### Run Deephaven Database
+
+### Run Deephaven Database (Python)
 
 From the `deephaven-core` directory, run
 ```
     docker-compose up
 ```
 This will start the database.  The console will fill with status and logging output.
 
-Killing the process (e.g. `Ctrl+C`) will stop Deephaven.  
+Killing the process (e.g. `Ctrl+C`) will stop Deephaven.
+
+### Run Deephaven Database (Groovy)
+
+From the `deephaven-core` directory, run
+```
+    docker compose --env-file .default_groovy.env up
```

Comment:
1) Have we switched everything to the `docker compose` syntax?  If so, I need to validate that everything works on all platforms, versions are the same, etc.

2) I have some concerns about the `.default_groovy.env` UX.  Pete and most other mac users can't find hidden files in finder, so making the env a hidden file will create a ton of heartache.  Something like `./env/default_groovy.env` would result in far less support pain.